snooty/diagnostics.py

```diff
@@ -944,3 +944,19 @@ def __init__(self) -> None:
             0,
             None,
         )
+
+
+class NestedDirective(Diagnostic):
+    severity = Diagnostic.Level.error
+
+    def __init__(
+        self,
+        nested_directive: str,
+        start: Union[int, Tuple[int, int]],
+        end: Union[None, int, Tuple[int, int]] = None,
+    ):
+        super().__init__(
+            f"""Nested directive detected: {nested_directive}.""",
```

Comment:
Could we make this error message more descriptive for writers? Perhaps along the lines of f"""Nesting of "{nested_directive}" directives prohibited"""

Reply:
i like the verbosity! updated